client/src/fluid/Fluid.ml

```diff
@@ -4162,8 +4162,34 @@ let rec updateKey
     | Keypress {key = K.ShiftEnter; _}, left, _ ->
         let doPipeline (astInfo : ASTInfo.t) : ASTInfo.t =
           let startPos, endPos = FluidUtil.getSelectionRange astInfo.state in
-          let topmostID = getTopmostSelectionID startPos endPos astInfo in
-          let findParent = startPos = endPos in
+          let topmostSelectionID =
+            getTopmostSelectionID startPos endPos astInfo
+          in
+          let topmostID, findParent =
```

Comment:
It seems like findParent is always `startPos = endPos`, so that could be simplified.

Reply:
There is one case where findParent is explicitly set to false (line 4178), when the token to the left is pipeable.

Reply:
Gotcha, thanks!